.buildkite/pipeline-resource-definitions/security-solution-quality-gate/kibana-serverless-security-solution-quality-gate-detection-engine.yml

```diff
@@ -2,7 +2,7 @@
 apiVersion: backstage.io/v1alpha1
 kind: Resource
 metadata:
-  name: buildkite-pipeline-kibana-serverless-security-solution-quality-gate-detection-engine
+  name: kb-serverless-security-solution-quality-gate-detection-engine
```

Comment:
@nuugen suggested that we do the kibana-buildkite prefix as a good practice, but it's true it's quite long/bloaty. Is there a legible alternative if we keep the kibana-buildkite, if not, I'm okay with kb-, we encode the categories in other fields anyway. @nuugen what do you think?

Reply:
kibana-buildkite with the rule of 36 chars makes the definition quite difficult.

We can either keep kb and the pipeline name as a convention OR `kibana-{abbreviation of team/org and scope}-{name}

Such as `kibana-serverless-ssqg-detection-engine`

What would you prefer?